releasenotes/notes/feat-support-langchain-0-1-0d8e0ddd6248c4ed.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    langchain:  This introduces support for ``langchain==0.1.0``. Note that this drops tracing support for deprecated
+    operations in ``langchain==0.1.0`` in favor of recommended alternative methods by [langchain](https://python.langchain.com/docs/changelog/core).
+    This includes:
+    - ``langchain.chain.Chain.run/arun`` with ``langchain.chain.Chain.invoke/ainvoke``
+    - ``langchain.embeddings.openai.OpenAIEmbeddings.embed_documents`` with ``langchain_openai.OpenAIEmbeddings.embed_documents``
+    - ``langchain.vectorstores.pinecone.Pinecone.similarity_search`` with ``langchain_pinecone.PineconeVectorStore.similarity_search``
```

Comment:
Can we move this to `ddtrace.contrib.langchain.__init__.py? We can then link the langchain docs here (l[ink to rst anchor docs](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor)). We do this in a few release notes so there should be examples